src/hooks/useWelcomeScreen.ts

```diff
@@ -29,6 +46,9 @@ export const useWelcomeScreen = (
   }, [cookie.welcomeScreenClosed, isMultisigSigner]);
 
   const updateState = (closed: boolean) => {
+    if (!enabled) {
```

Comment:
Is this check needed?

Reply:
It´s meant to ensure that state changes on the welcome-screen are only enabled if the theme supports the welcome-screen.